kpi/models/import_export_task.py

```diff
@@ -477,27 +488,26 @@ def export_upload_to(self, filename):
     return posixpath.join(self.user.username, 'exports', filename)
 
 
-class ProjectViewExportTask(ImportExportTask):
-    uid = KpiUidField(uid_prefix='pve')
-    result = PrivateFileField(upload_to=export_upload_to, max_length=380)
+class ExportTaskMixin:
+
+    @property
+    def default_email_subject(self) -> str:
+        return 'Report Complete'
+
+    def _get_export_details(self) -> tuple:
+        return self.data['type'], self.data['view']
 
     def _build_export_filename(
         self, export_type: str, username: str, view: str
     ) -> str:
         time = timezone.now().strftime('%Y-%m-%dT%H:%M:%SZ')
         return f'{export_type}-{username}-view_{view}-{time}.csv'
```

Comment:
Same as previous comment. This should be updated to allow an empty/None `view` parameter